documentation/dependencies/arch.dependencies

```diff
@@ -1,22 +1,18 @@
 bash
 git
 tar
-python-docutils
 pulseaudio
 libpulse
 dunst
-python-sphinx
 imagemagick
 graphviz
-python-virtualenv
 texlive-bin
 librsvg
 bzip2
 lzip
 lzop
 zstd
 xz
-python-pip
```

Comment:
Maybe I'm missing something, but if we remove `python-pip` how can you call `pip` in a fresh env that does not have `pip` installed by default?

Reply:
`python3 -m venv` will install pip inside the virtual env. So the host do not need to install it system-wide.